CHANGES.md

```diff
@@ -1,5 +1,13 @@
 # Change Log
 
+### 1.124.0
+
+#### @cesium/engine
+
+##### Fixes :wrench:
+
+- Handle uncaught promise rejection in `Skybox.update` that could cause tests to fail[#12307](https://github.com/CesiumGS/cesium/pull/12307)
```

Comment:
Unless the change affects how a developer would use the CesiumJS library, such as if a new function is added or a new type of error may be thrown, there's no need to mention it in the changelog.